docs-linting/markdownlint-custom-rules/codesnippet.js

```diff
@@ -4,6 +4,32 @@
 
 const common = require('./common');
 const detailStrings = require('./strings');
+const schemas = require('./schemas');
+const vscode = require('vscode');
+const { default: axios } = require('axios');
+
+// schema linting
+let allowedInterativeTypes;
+let allowedCodeSnippetAttributes;
+let codeSnippetDataResponse;
+
+function loadImageSchema() {
```

Comment:
Hi @bharney0 , should we change the function to loadCodeSnippetSchema or something similar?